lib/Voxa-Opearlo.js

```diff
@@ -6,57 +6,66 @@ const debug = require('debug')('voxa:opearlo');
 
 const defaultConfig = {
   initialState: 'entry',
-  opearloUserId: 'opearlo-user-id',
-  opearloAppName: 'opearlo-app-name',
-  opearloApiKey: 'opearlo-api-key',
+  userId: 'userId',
+  appName: 'appName',
+  apiKey: 'apiKey',
 };
 
 function register(skill, config) {
   const pluginConfig = _.merge({}, defaultConfig, config);
 
   skill.onSessionStarted((alexaEvent) => {
     OpearloAnalytics.initializeAnalytics(
-      pluginConfig.opearloUserId,
-      pluginConfig.opearloAppName,
+      pluginConfig.userId,
+      pluginConfig.appName,
       alexaEvent.session);
 
     debug('Session started');
   });
 
-  skill.onAfterStateChanged((alexaEvent, reply, transition) => {
+  skill.onIntentRequest((alexaEvent, reply) => {
     OpearloAnalytics.registerVoiceEvent(
       alexaEvent.session.user.userId,
-      transition.to,
+      'IntentRequest',
       alexaEvent.intent.name,
       alexaEvent.intent.params);
```

Comment:
if you look at the documentation the third parameter is the intent object, not the intent name

`  OpearloAnalytics.registerVoiceEvent(event.session.user.userId, "IntentRequest", event.request.intent);`